src/Persistence/MsApp/MsappArchive.cs

```diff
@@ -134,7 +134,7 @@ public MsappArchive(Stream stream, ZipArchiveMode mode, bool leaveOpen, Encoding
             foreach (var entry in ZipArchive.Entries)
             {
                 if (!canonicalEntries.TryAdd(NormalizePath(entry.FullName), entry))
-                    _logger?.LogInformation($"Duplicate entry found in archive: {entry.FullName}");
+                    _logger?.LogInformation("Duplicate entry found in archive: {EntryFullName}", entry.FullName);
```

Comment:
this will improve our structured logging, thank you for taking care of it!

Reply:
thank you to rules!